attributes/defaults.rb

```diff
@@ -54,6 +60,14 @@
   default['mysqld']['my.cnf']['mysqld_safe']['skip_log_error'] = true
 end
 
+# Password columns for user passwords
+default['mysqld']['pwd_col'] = case node['mysqld']['db_install']
```

Comment:
in order to hide this from the user completely, we could remove the attribute and move the decision for the proper pwd_col to the the password provider.
I think the user doesn't need to be able to change which column is actually used.